installation issue

## Description

I have a clean installation of ubuntu and I am trying to install and run the example. To have a good starting point, in order to build a beacon parcer.

I run:
```
sudo apt-get install cmake
sudo apt-get install ninja-build
sudo apt-get install build-essensials
git clone https://github.com/libcsp/libcsp.git
cmake -G Ninja -B builddir
cmake --build builddir
cmake --install builddir
```

and until here the steps implemented successfully.

when I trying to build examples using 
/buildall.py using cmake

i get
```
-- Build files have been written to: /home/panos/libcsp/examples/build
ninja: Entering directory `build
[1/9] Linking C executable csp_server
FAILED: csp_server 
: && /usr/bin/cc  -rdynamic CMakeFiles/csp_posix_helper.dir/csp_posix_helper.o CMakeFiles/csp_server.dir/csp_server.o -o csp_server  -lcsp  -lcsp_common  -lcsp_common && :
/usr/bin/ld: cannot find -lcsp_common: No such file or directory
/usr/bin/ld: cannot find -lcsp_common: No such file or directory
collect2: error: ld returned 1 exit status
[2/9] Linking C executable csp_server_client
FAILED: csp_server_client 
: && /usr/bin/cc  -rdynamic CMakeFiles/csp_posix_helper.dir/csp_posix_helper.o CMakeFiles/csp_server_client.dir/csp_server_client.o -o csp_server_client  -lcsp  -lcsp_common  -lcsp_common && :
/usr/bin/ld: cannot find -lcsp_common: No such file or directory
/usr/bin/ld: cannot find -lcsp_common: No such file or directory
collect2: error: ld returned 1 exit status
[3/9] Linking C executable csp_arch
FAILED: csp_arch 
: && /usr/bin/cc  -rdynamic CMakeFiles/csp_arch.dir/csp_arch.o -o csp_arch  -lcsp  -lcsp_common && :
/usr/bin/ld: cannot find -lcsp_common: No such file or directory
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
Traceback (most recent call last):
  File "/home/panos/libcsp/examples/./buildall.py", line 73, in <module>
    main(args.build_system)
  File "/home/panos/libcsp/examples/./buildall.py", line 61, in main
    build_with_cmake()
  File "/home/panos/libcsp/examples/./buildall.py", line 33, in build_with_cmake
    subprocess.check_call(cmake_compile)
  File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['ninja', '-C', 'build']' returned non-zero exit status 1.
```

moreover when thing to run the python_bindings_example_client.py I get no module name found libcsp_py3"

Is there any video about installation process or any step by step tutorial.

--- a/doc/INSTALL.md
+++ b/doc/INSTALL.md
@@ -88,12 +88,33 @@ use the following command:
 cmake --install builddir --component runtime
 ```
 
+### Building All Samples with CMake
+
 if you want to build tools and samples, define `CSP_BUILD_SAMPLES=ON`
 when you run `cmake`.
 
 ```shell
 cmake -B builddir -DCSP_BUILD_SAMPLES=ON
 ```
+
+### Python Bindings with CMake
+
+If you want to build Python bindings, define
+`CSP_ENABLE_PYTHON3_BINDINGS=ON` when you run `cmake`. You also need
+to enable the routing table (`CSP_USE_RTABLE`) when building the
+Python bindings.
+
+```shell
+cmake -B builddir -DCSP_ENABLE_PYTHON3_BINDINGS=ON -DCSP_USE_RTABLE=ON
+```
+
+To use the bindings, you need to install them to a location where
+Python searches by default or specify the path to Python:
+
+```
+PYTHONPATH=build python3 -c 'import libcsp_py3 as csp'
+```
+
 
 ## Reproducible Builds
 

